.openpublishing.redirection.json

```diff
@@ -3633,6 +3633,10 @@
       "redirect_url": "/dotnet/visual-basic/language-reference/xmldoc/index",
       "redirect_document_id": true
     },
+    {
+      "source_path": "docs/visual-basic/misc/",
+      "redirect_url": "/dotnet/visual-basic/language-reference/error-messages/"
+    },
```

Comment:
Will the redirection work for a whole folder like this ?

Reply:
Unfortunately not.